INSTALL

```diff
@@ -0,0 +1,120 @@
+#	Assetstore name: myAssetstore (example)
+#	Root directory: /assetstore
+# Girder’s REST API: http://localhost:8080/api/
+
+################################################################################
+# INSTALL APP-MAKING TOOLS
+################################################################################
+
+#-------------------------------------------------------------------------------
+# Install Android Studio
+# https://developer.android.com/studio/index.html
+#-------------------------------------------------------------------------------
+# - Open application
+# - Configure -> SDK Manager -> Launch Standalone SDK Manager
+# - Check correct Android version and install packages
+# - Android SDK License -> Accept License -> Install
+
+#-------------------------------------------------------------------------------
+# Install Java JDK
+# http://www.oracle.com/technetwork/java/javase/downloads
+#-------------------------------------------------------------------------------
+export JAVA_HOME="$(/System/Library/Frameworks/JavaVM.framework/Versions/Current/Commands/java_home)"
+
+#-------------------------------------------------------------------------------
+# Install NodeJS
+# https://nodejs.org/en/
+#-------------------------------------------------------------------------------
+sudo chown -R arno /usr/local/lib/node_modules/
+
+#-------------------------------------------------------------------------------
+# Install Ionic
+# https://ionicframework.com/getting-started/
+# Run sample app after installation: 
+#     ionic start --v2 myApp tabs; cd myApp; ionic serve
+#-------------------------------------------------------------------------------
+npm install -g ionic
+npm install -g cordova ios-sim ios-deploy
+ionic state reset
+
+#-------------------------------------------------------------------------------
+# Install Ionic-ResearchKit + MIT's VoiceUp app
+#-------------------------------------------------------------------------------
+git clone https://github.com/satra/MIT-VoiceUp-App.git
+
+#-------------------------------------------------------------------------------
+# Run app in local browser
+#-------------------------------------------------------------------------------
+# Edit MIT-VoiceUp-App/ionic/www/js/app.js:
+#	.constant('appConstants', {
+#  		'base_url': 'http://localhost:8080/api/v1/',
+ionic serve
+
+#-------------------------------------------------------------------------------
+# Share app to test in Ionic View (http://view.ionic.io):
+#-------------------------------------------------------------------------------
+ionic upload  # after creating ionic account
+ionic share EMAIL
+
+#-------------------------------------------------------------------------------
+# Package and run on a device or simulator
+#-------------------------------------------------------------------------------
+ionic upload  # after creating ionic account
+ionic package build android --profile dev
+cordova platform add android
```

Comment:
this step is unnecessary. the `ionic state reset` command earlier creates both the android and ios platforms.